roles/nova-common/templates/custom-filters/custom_disk_filter.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) 2012 OpenStack Foundation
```

Comment:
ues should make a phyton package - bbc-openstack-plugin .. We do something similar thing for glance. I will work with @cmt to port this to it's own pthon package.